src/seedsigner/views/psbt_views.py

```diff
@@ -446,11 +452,6 @@ def run(self):
             show_back_button=False,
         ).display()
 
-        # Clear out the bad PSBT
-        self.controller.psbt = None
-        self.controller.psbt_parser = None
-        self.controller.psbt_seed = None
-        
         return Destination(MainMenuView, clear_history=True)
```

Comment:
I'd just leave a reminder comment that `MainMenuView` will clear out the ephemeral psbt data for us.